lib/main-app.js

```diff
@@ -94,12 +99,12 @@ app.on('ready', function () {
 
   // Check update every day
   setInterval(function () {
-    checkUpdate()
+    if (!isDev) checkUpdate()
```

Comment:
Why do we have to re-check if the app is in development if we already check it in the `checkUpdate` function?

Reply:
This was a consideration I had but felt there may be some value in making it explicit for functions that call checkUpdate that it does not run in dev mode. Computationally it’s a lightweight conditional check but the concern was for future dodgers to have more clarity why the update check is not running in development mode. Feel free to remove.

Reply:
Well, I think you're right. Let's just leave it then.